tests/test_validation_data.py

```diff
@@ -0,0 +1,41 @@
+import pytest
+
+from anomaly_detector.anomaly_detector import AnomalyDetector
+import sys
+import logging
+
+from anomaly_detector.config import Configuration
+
+_LOGGER = logging.getLogger()
+_LOGGER.setLevel(logging.INFO)
+
+# create a logging format
+formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+
+sh = logging.StreamHandler(sys.stdout)
+sh.setFormatter(formatter)
+_LOGGER.addHandler(sh)
+
+w2v_logger = logging.getLogger('gensim.models')
+w2v_logger.setLevel(logging.ERROR)
```

Comment:
thats a lot of setup, I wonder if this could be done generally in the `Configuration` class. But ok...

Reply:
Your right. No need for all this logging setup. I'll strip it down.